go.mod

```diff
@@ -188,7 +188,7 @@ replace (
 	// Two changes
 	// (1) Testing infra
 	// (2) Fix bech32 bug
```

Comment:
```suggestion
```

Reply:
done (sorry had forgotten to push it up)